src/server/pfs/s3/s3_test.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+	"time"
+
+	minio "github.com/minio/minio-go"
+	"github.com/pachyderm/pachyderm/src/server/pfs/server"
+	"github.com/pachyderm/pachyderm/src/client/pkg/require"
+)
+
+func TestSimple(t *testing.T) {
+	pc := server.GetPachClient(t)
+	repo := "repo"
+	require.NoError(t, pc.CreateRepo(repo))
+	file := "file"
+	content := "content"
+	_, err := pc.PutFile(repo, "master", file, strings.NewReader(content))
+	require.NoError(t, err)
+	go func() { Serve(pc, 30655) }()
+	c, err := minio.NewWithRegion("127.0.0.1:30655", "id", "secret", false, "region")
```

Comment:
Btw, this will fail if you use `New` instead of `NewWithRegion` because the driver will send a request to try to figure out the region of the bucket which will fail because we don't implement it yet. I explicitly specified the region to get around this. You should get this working in the case where region isn't explicitly specified before merging it.